worldedit-bukkit/adapters/adapter-1_18/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_18_R1/PaperweightAdapter.java

```diff
@@ -186,7 +186,7 @@ public PaperweightAdapter() throws NoSuchFieldException, NoSuchMethodException {
         CraftServer.class.cast(Bukkit.getServer());
 
         int dataVersion = CraftMagicNumbers.INSTANCE.getDataVersion();
-        if (dataVersion != 2860) {
+        if (dataVersion != 2865) {
             throw new UnsupportedClassVersionError("Not 1.18!");
```

Comment:
```suggestion
        if (dataVersion != 2860 && dataVersion != 2865) {
            throw new UnsupportedClassVersionError("Not 1.18 or 1.18.1!");
```

In my opinion no need to ban 1.18, tho you should really use 1.18.1

Reply:
That's fair